tests/conftest.py

```diff
@@ -52,6 +52,27 @@ def empty_df():
     return pd.DataFrame([])
 
 
+@pytest.fixture(scope='session')
+def city_arrogance_split_dataset_and_model():
+    def city_leng(data):
+        data = data.copy()
+        data['city'] = data['city'].apply(len)
+        return data
+
+    df = pd.DataFrame(
+        {
+            'sex': [0, 1, 1, 0, 0, 1],
+            'city': ['ahhh', 'no', 'weeee', 'arg', 'eh', 'E'],
+            'arogance': [3, 1, 5, 2, 1, 1],
```

Comment:
```suggestion
            'arrogance': [3, 1, 5, 2, 1, 1],
```
Not that I have any clue where did you come up with this idea 🤔

Reply:
the human mind is an enigma